akka-stream/src/main/scala/akka/stream/impl/ActorRefSinkActor.scala

```diff
@@ -16,14 +16,15 @@ import akka.annotation.InternalApi
  * INTERNAL API
  */
 @InternalApi private[akka] object ActorRefSinkActor {
-  def props(ref: ActorRef, highWatermark: Int, onCompleteMessage: Any): Props =
-    Props(new ActorRefSinkActor(ref, highWatermark, onCompleteMessage))
+  def props(ref: ActorRef, highWatermark: Int, onCompleteMessage: Any, onFailureMessage: Throwable ⇒ Any): Props =
```

Comment:
we need to add this since otherwise typed won't be able to deliver a failed message